roles/ansible-runner/tasks/runner.yml

```diff
@@ -33,6 +33,12 @@
   secret_file_path: "{{ ansible_runner_secrets_path }}"
   secret_file_user: "{{ item.user }}"
 
+- name: Disable cron job
+  copy:
```

Comment:
state: touch will set a "changed" on every run, i though this way would be more idempotent.

The problem with removing the file when the runner is disabled is that if you log in to the box and touch the file to manually stop running the ci then on the next run the ci will remove it.